api/types/types.proto

```diff
@@ -1789,6 +1789,9 @@ message CertExtension {
 // RoleConditions is a set of conditions that must all match to be allowed or
 // denied access.
 message RoleConditions {
+    // IdemeumEntitlements is a list of entitlements set by idemeum.
+    repeated string IdemeumEntitlements = 1000 [ (gogoproto.jsontag) = "idemeum_entitlements,omitempty" ];
```

Comment:
Why should add the IdemeumEntitlements at end of all the attributes if required you can leave may be margin of 20 fields, currently it has 22 fields only, margin of 1000 is too much.

Reply:
Resolving issue as we discussed and dropped issue.